src/models/User.js

```diff
@@ -1,30 +1,82 @@
-const mongoose = require("mongoose");
+const mongoose = require('mongoose');
+const bcrypt = require('bcryptjs');
+const { adultValidator, adultNameFirstAndLast } = require('../utils/adultValidation');
+const {lettersOnlyValidator}= require("../utils/letterValidation.js");
 
-const UserSchema = new mongoose.Schema({
+// Define the Users schema
+const UsersSchema = new mongoose.Schema({
```

Comment:
It should be UserSchema since the model is User, and not Users.  Not a big deal.